### ISSUE TITLE
waitForIdle necessary on desktop where it is not on Android

### ISSUE THREAD

**Issue by tordjon** [JetBrains/compose-multiplatform#2427]
```
@Test
fun `increment x`() {
    var x: Int by mutableStateOf(0)

    rule.setContent {
        Text("$x", modifier = Modifier.clickable { x ++ })
    }

    rule.onNodeWithText("0").performClick()

//    rule.waitForIdle() // Necessary for synchronization
    rule.onNodeWithText("1").assertExists()
}
```

I must uncomment waitForIdle for this test to pass on desktop, but not on Android. Why is that? Is it intended?

This inconsistency makes it harder to share tests between the platforms

**Comment by m-sasha** [comment]
This has been fixed in the 1.5 release.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2427">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
